packages/emotion/package.json

```diff
@@ -6,7 +6,7 @@
   "type": "commonjs",
   "module": "./es/index.js",
   "main": "./lib/index.js",
-  "types": "./types/index.d.ts",
+  "types": "./types/global.d.ts",
```

Comment:
This is the new types entrypoint, `globals.d.ts` will have the correct reference to the emotion css prop type declaration and also will re-export everything from the `index.d.ts` file, which is the entry for the `emotion` package types.